Feature automatic reconnect

Tobis93 opened the pull request.

Tobis93 committed:
- the last connected device is saved in the internal Storage
- if there is a last connected device saved, the app tries to connect to this device when opening it
- added "search for other devices"-button to the right-top-menu. Click will start the Scan-Activity and a Scan

Tobis93 committed:
-removed "Connect to other device"-Button, because it is already present in another Branche

stekreis committed:
Merge branch 'master_lab' into feature_automatic_reconnect

anykeyjunkie commented on app/src/main/java/seemoo/fitbit/activities/MainActivity.java:
```diff
@@ -68,6 +68,7 @@ protected void onCreate(Bundle savedInstanceState) {
         initialize();
         requestPermissions();
         enableBluetooth();
+        checkCurrentDeviceIsSet();
```
lastDevice ist better

Tobis93 committed:
- Changed Current to Last for better understanding
- Relocated the Save-Last-Device-Functionality in the Method for Last-Devices-Listing

Tobis93 committed:
Merge branch 'master_lab' into feature_automatic_reconnect
